constellations/orion/src/tests_integration/account_integration_test.go

```diff
@@ -75,6 +75,34 @@ func Test_GetAccountById(t *testing.T) {
 	utils.ResetTable(t, domains.TABLE_ACCOUNTS)
 }
 
+// Test: Create 3 Accounts, GetNegativeBalances()
+func Test_GetNegativeBalanceAccounts(t *testing.T) {
+	// Create 3 accounts and 5 transactions, accounts 1 and 2 are negative
+	createAllAccounts(t)
+	createTransactions(t)
+
+	// Call endpoint
+	recorder := utils.SendHttpRequest(t, http.MethodGet, "/api/accounts/unpaid", nil)
+	assert.EqualValues(t, http.StatusOK, recorder.Code)
+
+	// Validate results
+	var accountSums []domains.AccountSum
+	if err := json.Unmarshal(recorder.Body.Bytes(), &accountSums); err != nil {
+		t.Errorf("unexpected error: %v\n", err)
+	}
+	assert.EqualValues(t, 1, accountSums[0].Id)
+	assert.EqualValues(t, "john_smith@example.com", accountSums[0].PrimaryEmail)
+	assert.EqualValues(t, "password1", accountSums[0].Password)
+	assert.EqualValues(t, -300, accountSums[0].Sum)
+	assert.EqualValues(t, 2, accountSums[1].Id)
+	assert.EqualValues(t, "bob_smith@example.com", accountSums[1].PrimaryEmail)
+	assert.EqualValues(t, "password2", accountSums[1].Password)
+	assert.EqualValues(t, -100, accountSums[1].Sum)
+
```

Comment:
Very nicely done testing. In addition, let's test `accountSums` is only size 2. There is NOT a third element in this array because account3 has non-negative account balance!